IMAGE:
  WIDTH: 640
  HEIGHT: 480
  ROI_START_HEIGHT: 340
  ROI_HEIGHT: 40
  SIGMA_GAUSSIANBLUR: 3

XYCAR:
  START_SPEED: 0.0
  MAX_SPEED: 5.0
  MIN_SPEED: 5.0
  SPEED_CONTROL_THRESHOLD: 15.0
  ACCELERATION_STEP: 0.1
  DECELERATION_STEP: 1.0
  CHOOSE_LANE_THRESHOLD: 15.0
  ERROR_MULTIPLICATION: 1.6

PID:
  P_GAIN: 0.30
  I_GAIN: 0.00
  D_GAIN: 0.08

MOVING_AVERAGE_FILTER:
  SAMPLE_SIZE: 10

TOPIC:
  PUB_NAME: /xycar_motor
  SUB_IMAGE_NAME: /usb_cam/image_raw/
  SUB_YOLO_NAME: /yolov3_trt_ros/detections
  QUEUE_SIZE: 1

CANNY:
  LOW_THRESHOLD: 150
  HIGH_THRESHOLD: 250

HOUGH:
  ABS_SLOPE_RANGE: 10.0
  THRESHOLD: 40
  MIN_LINE_LENGTH: 20
  MAX_LINE_GAP: 10

STOPLINE:
  STOPLINE_ROI_START_HIGHT: 300
  STOPLINE_ROI_START_ROW: 220
  STOPLINE_ROI_WIDTH: 220
  STOPLINE_ROI_HEIGHT: 20
  STOPLINE_THRESHOLD: 4
  STOPLINE_SLOPE_RANGE: 0.3
  KEEP_STOP_TIME: 5.0
  IGNORE_STOP_TIME: 2.0

TRAFFICSIGN:
  RESET_TIME: 5.0

DEBUG: true
